catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/resource/download/ReliableResourceDownloadManager.java

```diff
@@ -43,18 +42,19 @@
  */
 public class ReliableResourceDownloadManager {
 
-    static final int ONE_SECOND_IN_MS = 1000;
+    private static final int ONE_SECOND_IN_MS = 1000;
```

Comment:
Especially as this is private now, can we please get rid of it and just change the places using it to use `TimeUnit.SECONDS.toMillis(1)`?